tuntap-unix-linux.c

```diff
@@ -198,6 +202,7 @@ tuntap_sys_set_ipv6(struct device *dev, t_tun_in6_addr *s6, uint32_t bits) {
 int
 tuntap_sys_set_ifname(struct device *dev, const char *ifname, size_t len) {
 	struct ifreq ifr;
+	memset(&ifr, '\0', sizeof ifr);
```

Comment:
bug: last bytes is not initialized, caused to create device with name that is different from requested

Reply:
also prefix string with `(void) `